ui/pages/remote-mode/setup/setup-daily-allowance/remote-mode-setup-daily-allowance.component.tsx

```diff
@@ -109,24 +112,40 @@ export default function RemoteModeSetupDailyAllowance() {
   );
 
   const history = useHistory();
+  const location = useLocation();
+  const params = new URLSearchParams(location.search);
+  const delegationHash = params.get('delegationHash');
 
   const isRemoteModeEnabled = useSelector(getIsRemoteModeEnabled);
 
   const { assetsWithBalance } = useMultichainBalances();
 
+  const { enableRemoteMode, disableRemoteMode } = useRemoteMode({
```

Comment:
Same here, the `useRemoteMode` hook returns the `remoteModeConfig` object with the allowances. can we use that here to get the delegation? So we don't need to pass the delegation hash, and don't need to fetch from the selector. Maybe an update param like `?update=true` or something just to know when we need to update? or even, if theres a existing allowance when we enter the page, we can assume that the user wants to update, since will be only one allowance per account.

Reply:
As above 👍